server/cache.go

```diff
@@ -14,14 +14,12 @@
 package server
 
 import (
-	"bytes"
-	"math/rand"
+	"log"
 	"sync"
 	"time"
 
-	"github.com/golang/protobuf/proto"
+	"github.com/gogo/protobuf/proto"
```

Comment:
Actually, `goimports` do that for me, seems they are mostly the same, and we do use `gogo/protobuf` in kvproto, I think it's OK.